library/ordtree.lisp

```diff
@@ -41,6 +41,7 @@
 
   ;; unexported; a marker held by trees to enable self-balancing.
   (repr :enum)
+  (derive Eq)
```

Comment:
Point for future: (repr :enum) can be compared with `cl:eq`. Would be nice in the future for the deriver to know about this.